uber/templates/preregistration/preregbase.html

```diff
@@ -108,7 +116,9 @@
         {% endif %}
 
         {% for badge_type in c.BADGE_TYPE_PRICES %}
+
             {% if badge_type != c.ATTENDEE_BADGE %}
+        {% if (badge_type != c.SPONSOR_BADGE or c.SPONSOR_BADGE_AVAILABLE) and (badge_type != c.SHINY_BADGE or c.SHINY_BADGE_AVAILABLE) %}
```

Comment:
Whoops, this is definitely MFF-specific. Maybe it's fine though? I tested preregging badges and that seemed to work fine...

Reply:
Doesn't seem to break anything, so we'll keep it.